frontend/src/pages/shipments/ShipmentOffers.tsx

```diff
@@ -0,0 +1,121 @@
+import cx from 'classnames'
+import { FunctionComponent, useMemo } from 'react'
+import { Column, useSortBy, useTable } from 'react-table'
+import ButtonLink from '../../components/ButtonLink'
+import TableHeader from '../../components/table/TableHeader'
+import {
+  OffersForShipmentQuery,
+  useAllGroupsMinimalQuery,
+  useOffersForShipmentQuery,
+} from '../../types/api-types'
+import { offerCreateRoute } from '../../utils/routes'
+
+interface Props {
+  shipmentId: number
+}
+
+const ShipmentOffers: FunctionComponent<Props> = ({ shipmentId }) => {
```

Comment:
This is the Offers tabs:

<img width="1064" alt="Screen Shot 2021-03-19 at 3 46 43 PM" src="https://user-images.githubusercontent.com/3411183/111849244-55d92e80-88ca-11eb-935e-5d2654472b29.png">